reworked hifiasm/purge_dups, changed typos/wording, added merqury plots

abueg opened the pull request:
hello! 👋🏼 thank you all for your hard work on this tutorial, it'll be a valuable resource for running the workflows! 

my changes:

- reworked hifiasm/purge_dups section to flow more with the Hifiasm-HiC workflow, instead of the Hifiasm-pri/alt workflow
- added more merqury plots to explain how to find potential duplicates (I kept the "width: 80%" sizing for the image embedding used for the other merqury plot in the .md)
- removed the hands-on merqury box from the purge_dups section, as the workflows should be automatically running merqury..?
- typos/wording

bgruening commented:
@abueg thanks a lot. Can you rebase your branch against main. It seems there are some merge conflicts. Lets us know if you need help with that.

abueg commented:
hi @bgruening ! I have tried to rebase, I don't think I did it correctly, though? I'll put what I did here if that's ok:

```
(base) [linelle@blaziken ~/gits]$ cd training-material/
(base) [linelle@blaziken ~/gits/training-material]$ git remote add galaxyproject https://github.com/galaxyproject/training-material.git
(base) [linelle@blaziken ~/gits/training-material]$ git remote -v
galaxyproject	https://github.com/galaxyproject/training-material.git (fetch)
galaxyproject	https://github.com/galaxyproject/training-material.git (push)
origin	https://github.com/abueg/training-material.git (fetch)
origin	https://github.com/abueg/training-material.git (push)
upstream	https://github.com/galaxyproject/training-material.git (fetch)
upstream	https://github.com/galaxyproject/training-material.git (push)
(base) [linelle@blaziken ~/gits/training-material]$ git fetch galaxyproject
remote: Enumerating objects: 1049132, done.
remote: Counting objects: 100% (41862/41862), done.
remote: Total 1049132 (delta 41862), reused 41862 (delta 41862), pack-reused 1007270
Receiving objects: 100% (1049132/1049132), 1011.51 MiB | 21.52 MiB/s, done.
Resolving deltas: 100% (616490/616490), completed with 1176 local objects.
From https://github.com/galaxyproject/training-material
 * [new branch]              16x9                                         -> galaxyproject/16x9
 * [new branch]              BiodataAnalysisGroup-main                    -> galaxyproject/BiodataAnalysisGroup-main
 * [new branch]              RNA_family_model_construction                -> galaxyproject/RNA_family_model_construction
 * [new branch]              a11y                                         -> galaxyproject/a11y
 * [new branch]              add_atac-seq_tut                             -> galaxyproject/add_atac-seq_tut
 * [new branch]              anilthanki-GeneSeqToFamily_tutorial          -> galaxyproject/anilthanki-GeneSeqToFamily_tutorial
 * [new branch]              annotate-it                                  -> galaxyproject/annotate-it
 * [new branch]              anuprulez-patch-1                            -> galaxyproject/anuprulez-patch-1
 * [new branch]              anuprulez-patch-2                            -> galaxyproject/anuprulez-patch-2
 * [new branch]              assembly                                     -> galaxyproject/assembly
 * [new branch]              avans                                        -> galaxyproject/avans
 * [new branch]              blah                                         -> galaxyproject/blah
 * [new branch]              box-fix-biodiv                               -> galaxyproject/box-fix-biodiv
 * [new branch]              broken-box                                   -> galaxyproject/broken-box
 * [new branch]              bug-report                                   -> galaxyproject/bug-report
 * [new branch]              cd-box_identification_with_segmentation-fold -> galaxyproject/cd-box_identification_with_segmentation-fold
 * [new branch]              cheminf-wf-patch                             -> galaxyproject/cheminf-wf-patch
 * [new branch]              chip-seq                                     -> galaxyproject/chip-seq
 * [new branch]              citations-in-slides                          -> galaxyproject/citations-in-slides
 * [new branch]              comparative-metaomics-analysis               -> galaxyproject/comparative-metaomics-analysis
 * [new branch]              contributions                                -> galaxyproject/contributions
 * [new branch]              create-pull-request/patch                    -> galaxyproject/create-pull-request/patch
 * [new branch]              create-pull-request/patch-1624835582         -> galaxyproject/create-pull-request/patch-1624835582
 * [new branch]              create-pull-request/patch-1636931624         -> galaxyproject/create-pull-request/patch-1636931624
 * [new branch]              create-pull-request/patch-1639350792         -> galaxyproject/create-pull-request/patch-1639350792
 * [new branch]              create-pull-request/patch-1639955568         -> galaxyproject/create-pull-request/patch-1639955568
 * [new branch]              create-pull-request/patch-1646608525         -> galaxyproject/create-pull-request/patch-1646608525
 * [new branch]              css                                          -> galaxyproject/css
 * [new branch]              data_lib                                     -> galaxyproject/data_lib
 * [new branch]              decompose-ci                                 -> galaxyproject/decompose-ci
 * [new branch]              dexseq-tuto                                  -> galaxyproject/dexseq-tuto
 * [new branch]              disable_igv                                  -> galaxyproject/disable_igv
 * [new branch]              editly-ari                                   -> galaxyproject/editly-ari
 * [new branch]              embed-tests                                  -> galaxyproject/embed-tests
 * [new branch]              faster-videos                                -> galaxyproject/faster-videos
 * [new branch]              fix-build                                    -> galaxyproject/fix-build
 * [new branch]              fix-matrix                                   -> galaxyproject/fix-matrix
 * [new branch]              fix-news2matrix                              -> galaxyproject/fix-news2matrix
 * [new branch]              fix-pangeo-notebook                          -> galaxyproject/fix-pangeo-notebook
 * [new branch]              fix-path                                     -> galaxyproject/fix-path
 * [new branch]              fix-reports                                  -> galaxyproject/fix-reports
 * [new branch]              fix-search                                   -> galaxyproject/fix-search
 * [new branch]              fix-servers                                  -> galaxyproject/fix-servers
 * [new branch]              fix-tfs                                      -> galaxyproject/fix-tfs
 * [new branch]              format-tutorial-for-paper                    -> galaxyproject/format-tutorial-for-paper
 * [new branch]              gat-fixes-live                               -> galaxyproject/gat-fixes-live
 * [new branch]              gh-pages                                     -> galaxyproject/gh-pages
 * [new branch]              git-gat                                      -> galaxyproject/git-gat
 * [new branch]              gtn_jsa_gcms                                 -> galaxyproject/gtn_jsa_gcms
 * [new branch]              gtn_jsa_gcms-BACKUP                          -> galaxyproject/gtn_jsa_gcms-BACKUP
 * [new branch]              hexylena-patch-1                             -> galaxyproject/hexylena-patch-1
 * [new branch]              hexylena-patch-2                             -> galaxyproject/hexylena-patch-2
 * [new branch]              hrhotz-patch-1                               -> galaxyproject/hrhotz-patch-1
 * [new branch]              hrhotz-patch-4                               -> galaxyproject/hrhotz-patch-4
 * [new branch]              jennaj-ref-based-update                      -> galaxyproject/jennaj-ref-based-update
 * [new branch]              jobs-keep-going                              -> galaxyproject/jobs-keep-going
 * [new branch]              json-ld-asdf                                 -> galaxyproject/json-ld-asdf
 * [new branch]              main                                         -> galaxyproject/main
 * [new branch]              nekrut-patch-1                               -> galaxyproject/nekrut-patch-1
 * [new branch]              new-training                                 -> galaxyproject/new-training
 * [new branch]              nokogiri                                     -> galaxyproject/nokogiri
 * [new branch]              notebook-support                             -> galaxyproject/notebook-support
 * [new branch]              pangeo                                       -> galaxyproject/pangeo
 * [new branch]              progres                                      -> galaxyproject/progres
 * [new branch]              python-modular                               -> galaxyproject/python-modular
 * [new branch]              python-tutorials                             -> galaxyproject/python-tutorials
 * [new branch]              r-dplyr                                      -> galaxyproject/r-dplyr
 * [new branch]              review-comments                              -> galaxyproject/review-comments
 * [new branch]              reviewing-tutorial                           -> galaxyproject/reviewing-tutorial
 * [new branch]              rmd                                          -> galaxyproject/rmd
 * [new branch]              ruby_fix                                     -> galaxyproject/ruby_fix
 * [new branch]              rvm-update                                   -> galaxyproject/rvm-update
 * [new branch]              search-v2                                    -> galaxyproject/search-v2
 * [new branch]              separate-rdm-css                             -> galaxyproject/separate-rdm-css
 * [new branch]              simonbray/ctb                                -> galaxyproject/simonbray/ctb
 * [new branch]              smaller-hof                                  -> galaxyproject/smaller-hof
 * [new branch]              snippet_doc                                  -> galaxyproject/snippet_doc
 * [new branch]              spanscriptomics-emergencia                   -> galaxyproject/spanscriptomics-emergencia
 * [new branch]              subinamehta-patch-1                          -> galaxyproject/subinamehta-patch-1
 * [new branch]              subinamehta-patch-2                          -> galaxyproject/subinamehta-patch-2
 * [new branch]              subinamehta-patch-3                          -> galaxyproject/subinamehta-patch-3
 * [new branch]              subinamehta-sgpsfixing                       -> galaxyproject/subinamehta-sgpsfixing
 * [new branch]              tapas-banner                                 -> galaxyproject/tapas-banner
 * [new branch]              theme-updated                                -> galaxyproject/theme-updated
 * [new branch]              timing-sars-cov-2-tut                        -> galaxyproject/timing-sars-cov-2-tut
 * [new branch]              topic/amplicon-2019                          -> galaxyproject/topic/amplicon-2019
 * [new branch]              training-material-intro                      -> galaxyproject/training-material-intro
 * [new branch]              trinity                                      -> galaxyproject/trinity
 * [new branch]              unaligned-ncrnas-analysis                    -> galaxyproject/unaligned-ncrnas-analysis
 * [new branch]              update-workers                               -> galaxyproject/update-workers
 * [new branch]              vid-issues                                   -> galaxyproject/vid-issues
 * [new branch]              video-timeout                                -> galaxyproject/video-timeout
 * [new branch]              yvanlebras-patch-1                           -> galaxyproject/yvanlebras-patch-1
 * [new tag]                 2019-7-16                                    -> 2019-7-16
 * [new tag]                 2021-02-01                                   -> 2021-02-01
 * [new tag]                 2021-03-01                                   -> 2021-03-01
 * [new tag]                 2021-04-01                                   -> 2021-04-01
 * [new tag]                 2021-05-01                                   -> 2021-05-01
 * [new tag]                 2021-06-01                                   -> 2021-06-01
 * [new tag]                 2021-07-01                                   -> 2021-07-01
 * [new tag]                 2021-08-01                                   -> 2021-08-01
 * [new tag]                 2021-09-01                                   -> 2021-09-01
 * [new tag]                 2021-10-01                                   -> 2021-10-01
 * [new tag]                 2021-11-01                                   -> 2021-11-01
 * [new tag]                 2021-12-01                                   -> 2021-12-01
 * [new tag]                 2022-01-01                                   -> 2022-01-01
 * [new tag]                 2022-02-01                                   -> 2022-02-01
 * [new tag]                 2022-03-01                                   -> 2022-03-01
 * [new tag]                 2022-04-01                                   -> 2022-04-01
(base) [linelle@blaziken ~/gits/training-material]$ git rebase master
error: cannot rebase: You have unstaged changes.
error: Please commit or stash them.
(base) [linelle@blaziken ~/gits/training-material]$ git status
On branch master
Your branch is up to date with 'origin/master'.

Changes not staged for commit:
  (use "git add <file>..." to update what will be committed)
  (use "git restore <file>..." to discard changes in working directory)
	modified:   topics/assembly/tutorials/vgp_workflow_training/tutorial.md

Untracked files:
  (use "git add <file>..." to include in what will be committed)
	topics/assembly/images/vgp_assembly/yeast_p_merqury_hap2.png

no changes added to commit (use "git add" and/or "git commit -a")
(base) [linelle@blaziken ~/gits/training-material]$ git add topics/assembly/tutorials/vgp_workflow_training/tutorial.md
(base) [linelle@blaziken ~/gits/training-material]$ git add topics/assembly/images/vgp_assembly/yeast_p_merqury_hap2.png
(base) [linelle@blaziken ~/gits/training-material]$ git status
On branch master
Your branch is up to date with 'origin/master'.

Changes to be committed:
  (use "git restore --staged <file>..." to unstage)
	new file:   topics/assembly/images/vgp_assembly/yeast_p_merqury_hap2.png
	modified:   topics/assembly/tutorials/vgp_workflow_training/tutorial.md

(base) [linelle@blaziken ~/gits/training-material]$ git rebase master
error: cannot rebase: Your index contains uncommitted changes.
error: Please commit or stash them.
(base) [linelle@blaziken ~/gits/training-material]$ git stash
Saved working directory and index state WIP on master: 3a5a008789f added pre-purge merqury plots
(base) [linelle@blaziken ~/gits/training-material]$ git rebase master
Current branch master is up to date.
(base) [linelle@blaziken ~/gits/training-material]$ git status
On branch master
Your branch is up to date with 'origin/master'.

nothing to commit, working tree clean
```

bgruening commented:
Try to please `git rebase main`. We renamed the main/master branch to main. A recommendation is to name the galaxyproject remote `upstream`. This is a convention to indicate which project is the original or authority in the end. Hope that helps.

gallardoalba commented on topics/assembly/tutorials/vgp_workflow_training/tutorial.md:
```diff
@@ -58,19 +58,19 @@ This tutorial assumes you are comfortable getting data into Galaxy, running jobs
 # VGP assembly workflow structure
 
-The VGP assembly pipeline has a modular organization, consisting in five main subworkflows (fig. 1), each one integrated by a series of data manipulation steps. Firstly, it allows the evaluation of intermediate steps, which facilitates the modification of parameters if necessary, without the need to start from the initial stage. Secondly, it allows to adapt the workflow to the available data (e.g ...
+The VGP assembly pipeline has a modular organization, consisting of five main subworkflows (fig. 1), each one integrated by a series of data manipulation steps. This modular nature allows for the quality control assessment of intermediate steps, so that one can modify parameters (if necessary) without having to start the assembly from scratch. Additionally, this allows one to adapt the pipeline t ...
```
```suggestion
The VGP assembly pipeline has a modular organization, consisting of five main sub-workflows (fig. 1), each one integrated by a series of data manipulation steps. This modular nature allows for the quality control assessment of intermediate steps, so that one can modify parameters (if necessary) without having to start the assembly from scratch. Additionally, this allows one to adapt the pipeline to the available data.
```

gallardoalba replied:
The last sentence was removed because it was a bit redundant.

gallardoalba commented on topics/assembly/tutorials/vgp_workflow_training/tutorial.md:
```diff
@@ -175,20 +175,24 @@ Now that our data and workflows are imported, we can run our first workflow. Bef
 
 
-# HiFi phased assembly with hifiasm
+# HiC-phased assembly with hifiasm
```
```suggestion
# HiFi phased assembly with hifiasm
```

gallardoalba replied:
It shouldn't be modified I think.

gallardoalba commented on topics/assembly/tutorials/vgp_workflow_training/tutorial.md:
```diff
@@ -175,20 +175,24 @@ Now that our data and workflows are imported, we can run our first workflow. Bef
+# HiC-phased assembly with hifiasm
 
-After the genome profiling, the next step is to run the **VGP HiFi phased assembly with hifiasm and HiC data workflow**. This workflow uses **hifiasm** to generate initial primary and alternate pseudohaplotype assemblies. In addition, this workflow includes three tools for evaluating the assembly completeness: **QUAST**, **BUSCO** and **Merqury**.
+After genome profiling, the next step is to run the **VGP HiC-phased assembly with hifiasm and HiC data workflow**. This workflow uses **hifiasm** (HiC mode) to generate HiC-phased haplotypes (hap1 and hap2). This is in contrast to its default mode, which generates primary and alternate pseudohaplotype assemblies. This workflow includes three tools for evaluating assembly quality: **QUAST**, **BU ...
```
```suggestion
After genome profiling, the next step is to run the **VGP HiFi phased assembly with hifiasm and HiC data workflow**. This workflow uses **hifiasm** (HiC mode) to generate HiC-phased haplotypes (hap1 and hap2). This is in contrast to its default mode, which generates primary and alternate pseudohaplotype assemblies. This workflow includes three tools for evaluating assembly quality: **QUAST**, **BUSCO** and **Merqury**.
```

gallardoalba commented on topics/assembly/tutorials/vgp_workflow_training/tutorial.md:
```diff
@@ -293,9 +293,9 @@ This workflow generates a large number of outputs, among which we should highlig
+Once the assemblies generated by **hifiasm** have been purged, the next step is to run the **VGP hybrid scaffolding with Bionano optical maps workflow**. It will integrate the information provided by optical maps with primary assembly sequences in order to detect and correct structural variants, such as chimeric joins and misoriented contigs. In addition, this workflow includes some additonal ste ...
 
-> ### {% icon hands_on %} Hands-on: VGP hybrid scaffolding with Bionano optical maps workflow
+> ### {% icon hands_on %} Hands-on: VGP scaffolding with Bionano optical maps workflow
```
```suggestion
> ### {% icon hands_on %} Hands-on: VGP hybrid scaffolding with Bionano optical maps workflow
```

gallardoalba replied:
I would keep "hybrid".

gallardoalba commented on topics/assembly/tutorials/vgp_workflow_training/tutorial.md:
```diff
@@ -304,37 +304,37 @@ Once the assemblies generated by **hifiasm** has been purged, the next step is t
     
 
-# Hybrid scaffolding with Hi-C data
+# Scaffolding with HiC data
```
```suggestion
# Hybrid scaffolding with Hi-C data
```

gallardoalba replied:
I would keep the original title.

gallardoalba commented:
This PR will fixes the broken link https://github.com/abueg/training-material/pull/1

gallardoalba reviewed (commented):
Thank you for your suggestions! This PR fixes the broken link https://github.com/abueg/training-material/pull/1

abueg commented:
checked the figure numbers!

gallardoalba commented:
> checked the figure numbers!

Also this https://github.com/abueg/training-material/pull/2

gallardoalba reviewed (approved):
Thank u!
